lib/virt_autotest/utils.pm

```diff
@@ -187,7 +187,7 @@ sub is_hyperv_virtualization {
 
 # Return 1 if it is SLES MU virt test, otherwise return 0
 sub is_sles_mu_virt_test {
-    return is_sle && get_var('REGRESSION', '') =~ /xen|kvm|qemu|hyperv|vmware/ && !get_var("VIRT_AUTOTEST");
+    return is_sle && get_var('REGRESSION', '') =~ /xen|kvm|qemu|hyperv|vmware|kubevirt/ && !get_var("VIRT_AUTOTEST");
```

Comment:
Hi @tbaev , by design REGRESSION is kind of the hypervisor type, although not well designed from beginning(eg kvm, qemu both exists), anyway kubevirt does not fit this way here. How about set REGRESSION=kvm for kubevirt tests in medium? Will it meet the need here?